flow-typed/debugger-html.js

```diff
@@ -222,6 +222,32 @@ declare module "debugger-html" {
   };
 
   /**
+   * SourceScope
+   * @memberof types
+   * @static
+   */
+  declare type SourceScope = {
+    type: string,
+    start: Location,
+    end: Location,
+    bindings: {
+      [name: string]: Location[]
+    }
+  };
+
+  /*
+   * MappedScopeBindings
+   * @memberof types
+   * @static
+   */
+  declare type MappedScopeBindings = {
```

Comment:
These don't seem to be used.

Reply:
I pulled it in because it will be used by `devtools-map-bindings` shortly and seemed like a dependency of pulling in the library.

Reply:
Actually, now that I say that, it was just pulled in 10 minutes ago by another PR that uses it.